taxi/models.py

```diff
@@ -0,0 +1,30 @@
+from tkinter.constants import CASCADE

```

Comment:
The import `from tkinter.constants import CASCADE` is unnecessary and should be removed. The `CASCADE` option for the `ForeignKey` is already imported from `django.db.models`.